core/tests/performance_framework/perf_domain.py

```diff
@@ -144,3 +144,36 @@ def get_dom_ready_time_millisecs(self):
     def get_request_time_millisecs(self):
         """Returns the total request time (in milliseconds)."""
         return self._get_duration_millisecs('responseEnd', 'requestStart')
+
+
+class MultiplePageSessionMetrics(object):
+    """Domain object for multiple PageSessionMetrics to provide average
+    metrics, so as to reduce the variation between statistics obtained during
+    different page load sessions. This may happen due to various factors like
+    background processes.
+    """
+
+    def  __init__(self, page_session_metrics):
+        self.page_metrics = page_session_metrics
+        self._validate()
+
+    def _validate(self):
+        if not isinstance(self.page_metrics, list):
+            raise utils.ValidationError(
+                'Expected page_session_metrics to be a list, '
+                'received %s' % self.page_metrics)
+
+    def get_average_page_load_time_millisecs(self):
+        """Returns the average total page load time (in milliseconds)."""
+        return (sum(item.get_page_load_time_millisecs()
+                    for item in self.page_metrics)) / len(self.page_metrics)
+
+    def get_average_dom_ready_time_millisecs(self):
+        """Returns the average total page load time (in milliseconds)."""
```

Comment:
I missed this the first time. Copy paste error? I suspect the docstring should be referring to DOM ready time.